include/openthread/platform/ble.h

```diff
@@ -281,6 +315,21 @@ otError otPlatBleGattServerIndicate(otInstance *aInstance, uint16_t aHandle, con
  */
 extern void otPlatBleGattServerOnWriteRequest(otInstance *aInstance, uint16_t aHandle, const otBleRadioPacket *aPacket);
 
+/**
+ * Function to retrieve from platform BLE link capabilities.
+ *
+ * @param[in] aInstance             The OpenThread instance structure.
+ *
+ */
+otBleLinkCapabilities otPlatGetBleLinkCapabilities(otInstance *aInstance);
```

Comment:
Rename to follow name style of using `otPlatBe` as a prefix?  The general format for all platform APIs is `otPlat<ModeulName><API Name>`

`otPlatGetBleLinkCapabilities` -> `otPlatBleGetLinkCapabilities`

Reply:
fixed.